benchmarks/lib/build.gradle.kts

```diff
@@ -0,0 +1,68 @@
+val accpVersion: String? by project
+val accpLocalJar: String by project
+val fips: Boolean by project
+
+plugins {
+    `java-library`
+    id("me.champeau.jmh") version "0.7.1"
+    id("io.morethan.jmhreport") version "0.9.0"
+    id("com.google.osdetector") version "1.7.0"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    jmh("org.projectlombok:lombok:1.18.28")
+    jmh("org.projectlombok:lombok:1.18.28")
+    jmh("org.bouncycastle:bcprov-jdk15on:1.70")
+
+    val accpArtifactId =
+    if (project.hasProperty("fips"))
+        "AmazonCorrettoCryptoProvider-FIPS"
+    else
+        "AmazonCorrettoCryptoProvider"
+
+    val publishedAccpVersion =
+    if (project.hasProperty("accpVersion"))
+        accpVersion
+    else
+        "2.3.1"
```

Comment:
do we want to just always take the latest instead of fixing this to a specific version we'll have to remember to update through time?

Reply:
Replaced "2.3.1" with "2.+" so it would use the latest.